man/en/sos-upload.1

```diff
@@ -0,0 +1,111 @@
+.TH SOS UPLOAD 1 "July 2024"
```

Comment:
`lintian` will bring this as an issue with `wrong-manual-section` as per PR #3703. So, this ideally needs to be `UPLOAD` and not `SOS UPLOAD`